### ISSUE TITLE
Update acknowledgments section

### ISSUE THREAD

**Issue by geordie666** [legacysurvey/legacysurvey#157]
At the minimum, the additional BASS grant number needs to be added to read "...Chinese
National Natural Science Foundation (Grant #12120101003, #11433005)" (we currently only list #11433005).

We're likely adding a generic "short" acknowledgement, too, though. So, hold off on changing this until that's finalized.

**Comment by geordie666** [comment]
The short acknowledgment — which should only be used when _also_ including the [standard DESI acknowledgment](https://desi.lbl.gov/trac/wiki/GuidetoPub/DESIAcknowledgements) is:

```
The DESI Legacy Imaging Surveys consist of three individual and complementary projects: the Dark Energy Camera Legacy Survey (DECaLS), the Beijing-Arizona Sky Survey (BASS), and the Mayall z-band Legacy Survey (MzLS). DECaLS, BASS and MzLS together include data obtained, respectively, at the Blanco telescope, Cerro Tololo Inter-American Observatory, NSF’s NOIRLab; the Bok telescope, Steward Observatory, University of Arizona; and the Mayall telescope, Kitt Peak National Observatory, NOIRLab. NOIRLab is operated by the Association of Universities for Research in Astronomy (AURA) under a cooperative agreement with the National Science Foundation. Legacy Surveys also uses data products from the Near-Earth Object Wide-field Infrared Survey Explorer (NEOWISE), a project of the Jet Propulsion Laboratory/California Institute of Technology, funded by the National Aeronautics and Space Administration. Legacy Surveys was supported by: the Director, Office of Science, Office of High Energy Physics of the U.S. Department of Energy; the National Energy Research Scientific Computing Center, a DOE Office of Science User Facility;  the U.S. National Science Foundation, Division of Astronomical Sciences; the National Astronomical Observatories of China, the Chinese Academy of Sciences and the Chinese National Natural Science Foundation. The complete acknowledgments can be found at https://www.legacysurvey.org/acknowledgment/.
```

**Comment by weaverba137** [comment]
@geordie666, is the "short acknowledgement" above intended to replace the _entire_ paragraph that begins "The DESI Legacy Imaging Surveys..." in `acknowledgment.rst`?

**Comment by geordie666** [comment]
The short acknowledgment is meant to replace the entire section under: 

`When using data from the Legacy Surveys in papers, please use the following acknowledgment:` 

that begins: 

`The Legacy Surveys consist of three individual and complementary projects: the Dark Energy Camera Legacy Survey (DECaLS; Proposal ID #2014B-0404; PIs: David Schlegel and Arjun Dey), the Beijing-Arizona Sky Survey ...`

and concludes:

`... The Legacy Surveys imaging of the DESI footprint is supported by the Director, Office of Science, Office of High Energy Physics of the U.S. Department of Energy under Contract No. DE-AC02-05CH1123, by the National Energy Research Scientific Computing Center, a DOE Office of Science User Facility under the same contract; and by the U.S. National Science Foundation, Division of Astronomical Sciences under Contract No. AST-0950945 to NOAO.`

**_But only in the case_** where the [standard DESI acknowledgment](https://desi.lbl.gov/trac/wiki/GuidetoPub/DESIAcknowledgements) is already being used as part of the acknowledgments. So, the "short acknowledgment" needs to be added in a separate section entitled something similar to "Short acknowledgment for DESI collaborators" that notes that the entire text in the **Scientific Publication Acknowledgment** section can be replaced with the following shorter acknowledgment _when the standard DESI acknowledgment is also present_.

If that's still not clear, which is entirely possible, see how the acknowledgment [in the desitarget paper ](https://desi.lbl.gov/DocDB/cgi-bin/private/ShowDocument?docid=6693) is structured, as an example.

**Comment by weaverba137** [comment]
OK, I think I understand that, but I'll let you look at the final product.

**Comment by weaverba137** [comment]
Closed by #162.
